homeassistant/components/homewizard/sensor.py

```diff
@@ -349,6 +354,7 @@ class HomeWizardSensorEntityDescription(
         device_class=SensorDeviceClass.GAS,
         state_class=SensorStateClass.TOTAL_INCREASING,
         value_fn=lambda data: data.total_gas_m3,
+        attr_fn=lambda data: {ATTR_LAST_UPDATE: data.gas_timestamp},
```

Comment:
Gas measurement changes about every 5 minutes. Would it not be better to have a seperate sensor. It could be disables by default.

Reply:
A pre-disabled sensor would do, but it was 'rejected' on initial release → https://github.com/home-assistant/core/pull/55812#discussion_r776924496

For the user there is little or no extra information, other than knowing the _exact_ timestamp. For most users the 'Last updated' will be sufficient.

Reply:
Right, using state attributes is even less appreciated AFAIN.

Reply:
IMO it makes sense to have this info.

Reply:
> A pre-disabled sensor would do, but it was 'rejected' on initial release → [#55812 (comment)](https://github.com/home-assistant/core/pull/55812#discussion_r776924496)

I think that comment was about what this provided instead of the entities normal `last_updated`. I suppose this is a more accurate fetch of when the number was retrieved rather than when the entity updated.

This should still be a sensor and not an attribute and I think disabled by default.